packages/toolpad-app/src/runtime/AppLayout.tsx

```diff
@@ -25,20 +38,29 @@ export interface NavigationEntry {
 const DRAWER_WIDTH = 250; // px
 
 interface AppPagesNavigationProps {
-  activePage?: string;
+  activePageSlug?: string;
   pages: NavigationEntry[];
   clipped?: boolean;
   search?: string;
 }
 
 function AppPagesNavigation({
-  activePage,
+  activePageSlug,
   pages,
   clipped = false,
   search,
 }: AppPagesNavigationProps) {
   const navListSubheaderId = React.useId();
 
+  const theme = useTheme();
+
+  const productIcon = new URL(
```

Comment:
Does anyone know why I can't import images in the runtime like the editor does in its header to show the logo?
This solution works in dev but in a production app it doesn't. Importing the files directly doesn't work in dev for the runtime.

Reply:
Yes,

* When runtime gets build, the svg gets copied to the dist folder of the runtime, the import gets rewritten to include the content hash.
* When vite imports the runtime, it doesn't seem to do its [asset handling](https://vitejs.dev/guide/assets) on this file.

Let's not block this PR on the problem. We can move this PR forward without the Toolpad icon

Reply:
Looks like the same problem as https://github.com/vitejs/vite/issues/11943

Reply:
Actually, looked a bit deeper into it. It looks like esbuild translates the asset into a path and copies the file alongside the compiled output. Instead I externalized imports that look like assets so that esbuild preserves the import. Now it seems that vite picks up this import and translates it correctly to an asset.

Reply:
Awesome, thanks a lot, that was a though one!
Ok, if all checks are green which they should be now I will mark this PR for review.